tests/mir-opt/pre-codegen/checked_ops.rs

```diff
@@ -1,5 +1,5 @@
 // skip-filecheck
-//@ compile-flags: -O -Zmir-opt-level=2 -Cdebuginfo=2
```

Comment:
why this change?

Reply:
I don't mind it, either way, tbh, I just wanted to know the reasoning so that I know it wasn't Perfectly Random.